packages/components/src/components/link-pure/link-pure-styles.ts

```diff
@@ -63,6 +63,12 @@ export const getComponentCss = (
               '&(a:focus:not(:focus-visible))::before': {
                 border: 0,
               },
+              // revert injected hover styles from InitialStyles partial
+              '&(a:hover)': {
```

Comment:
That doesn't seem right and if really needed should be wrapped in the hover media query.